tests/test_accelerator.py

```diff
@@ -160,3 +162,57 @@ def load_config(models, input_dir):
 
             # mode.class_name is NOT loaded from config
             self.assertTrue(model.class_name != model.__class__.__name__)
+
+    @slow
+    def test_accelerator_bnb(self):
+        """Tests that the accelerator can be used with the BNB library."""
+        from transformers import AutoModelForCausalLM
+
+        model = AutoModelForCausalLM.from_pretrained(
+            "EleutherAI/gpt-neo-125m",
+            load_in_8bit=True,
+            device_map="auto",
+        )
+        accelerator = Accelerator()
+
+        # This should work
+        model = accelerator.prepare(model)
+
+        with init_empty_weights():
+            model = AutoModelForCausalLM.from_pretrained(
+                "EleutherAI/gpt-neo-125m",
+            )
+            device_map = infer_auto_device_map(model)
+            device_map["lm_head"] = "cpu"
+
+        model = AutoModelForCausalLM.from_pretrained(
+            "EleutherAI/gpt-neo-125m", device_map=device_map, load_in_8bit=True, llm_int8_enable_fp32_cpu_offload=True
+        )
```

Comment:
@sgugger I think this will fail as the docker image is not using the `main` branch of `transformers` no?

Reply:
happy to skip it until the next release of `transformers`

Reply:
Yes we are not installing from source.